### ISSUE TITLE
Garbage Collection

### ISSUE THREAD

**Issue by Dominilk** [osa1/mincaml#6]
Hi!

Just in upfront: Yes I read the README haha
but I think cranelift does support stack maps now (I asked on the rust discord and I think some members mentioned that stackmaps were supported). I think it would be a great addition to introduce garbage collection as it is an awesome example on how to use cranelift!

Thanks,
Dominik

**Comment by osa1** [comment]
Yeah I think they implemented stack maps in cranelift while implementing Wasm reference types for wasmtime.

I agree that it would be great addition to the compiler. I'm a bit busy these days but I'll see if I can allocate some time for this project.
